ecommerce/src/main/resources/application.yml

```
@@ -0,0 +1,47 @@
+spring:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)